softwerkskammer/lib/mailsender/message.js

```diff
@@ -14,6 +14,7 @@ class Message {
     if (body && member) {
       this.setSubject(body.subject);
       this.setMarkdown(body.markdown);
+      this.sender = member;
```

Comment:
Ich versteh immer noch nicht, warum du hier den ganzen member brauchst

Reply:
Prinzipiell würde das auch nur mit der Mail-Adresse gehen bzw. man könnte sich aus Mail-Adresse und Display-Name wieder was Member-artiges bauen, aber allgemein, funktioniert der Mail-Sende-Code mit Member, zumindest sieht es für mich so aus, man holt sich ja auch für die Gruppen die Member und verwendet die dann um die Message entsprechend mit BCC zu versehen.

Reply:
Ich finde aber den Weg sich aus Mail-Adresse und potentiell Name wieder so ein Member-artiges Objekt zu bauen recht hässlich. Wenn man den Namen weglässt, dann wird potentiell ein "null null" in der Mail als Empfänger-Klarnamen verwendet, s. meinen anderen Kommentar:
> Ich habe mal geschaut: prinzipiell würde gehen, wenn man den Sender aus der senderAddress in der message rekonstruiert. Dann gibt displayName() aber "null null" zurück.

Reply:
wo brauchst Du den displayName?

Reply:
Der wird vom Mail-Sende-Code verwendet, sieht vielleicht etwas komisch aus, wenn man eine Mail kommt mit "null null <mail@adresse.de>" als Empfänger?

Reply:
Aber streng genommen braucht man den nicht, das stimmt.